BEIMA.Client/src/pages/DeviceTypes/DeviceTypePage.js

```diff
@@ -172,9 +151,24 @@ const DeviceTypePage = () => {
       setDeletedFields([...item.deletedFields])
     }
 
+    const attemptDeleteType = async (id) => {
+      let response = await deleteDeviceType(id);
```

Comment:
I think we should add in an "are you sure" prompt before deleting as well, in case they accidentally click it or something.

Reply:
I added a simple alert for now, but I created #333 to handle deletion error/warning messages for devices and device types